### DISCUSSION TITLE
Docker registry variable

### DISCUSSION THREAD

**Discussion by secsonthebeach** [e-m-b-a/emba#637] [Ideas]
Hello,
Could you add an installation configuration variable to allow for the specification of a private container, proxy-cache registry that would point to the emba image?

Instead of docker attempting to run “embeddedanalyzer/emba” it would run “private-registry.harbor.domain/dockerhub.reference/embeddedanalyzer/emba”

**Comment by m-1-k-3** [comment]
good idea. I will take a look

**Comment by m-1-k-3** [comment]
see here https://github.com/e-m-b-a/emba/wiki/Installation#classic-installation
